test/functional/web-worker/test-amp-worker.js

```diff
@@ -39,15 +40,16 @@ describe('invokeWebWorker', () => {
     const fakeWorkerClass = () => fakeWorker;
     fakeWin = {Worker: fakeWorkerClass};
 
-    toggleExperiment(fakeWin, 'web-worker', true);
+    isExperimentOnStub = sandbox.stub(experiments, 'isExperimentOn');
```

Comment:
This is supppppperrrrr icky, and not valid ES6. `import * as experiments` means `experiments` is not an mutable object.

Why was this changed?